.gitlab-ci.yml

```diff
@@ -109,6 +109,7 @@ OSTree:
           # - openstack/fedora-34-x86_64
           - openstack/rhel-8-x86_64
           - openstack/rhel-8.4-x86_64
+          - openstack/rhel-8.5-x86_64
```

Comment:
@achilleas-k I propose to replace the 8.4 runner with 8.5 instead, not add another one. Also once the runner PR lands I prefer to actually change all of the 8.4 runners (we can do that separately).

Reply:
Changed ostree tests to run in 8.5 and the rest in 8.4 until we support all image types in 8.5.